src/lib/components/Search.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { createEventDispatcher } from "svelte";
+
+    let searchTerm = '';
+
+    const dispatch = createEventDispatcher();
+
+    function onSubmit(){
+        dispatch("search", {searchTerm})
+    }
+
+</script>
+
+<form on:submit|preventDefault={onSubmit} class="p-4 space-y-4">
```

Comment:
role search seems appropriate here https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/search.html

Reply:
Add role search to form and input tags for Search.svelte